README.md

```diff
@@ -247,6 +247,18 @@ function composePageUrl(pageName, pageId) {
 
 > See also [get](#get).
 
+### `create`
+
+Creates a new object. 
+
+```js
+function createPage(pageName) {
+  return new Page(pageName)
```

Comment:
the new keyword already implies “create”. This seems potentially confusing, since it’s  adds no meaningful abstraction.

would it make sense for this example to be like delete (where the example assumes a database model?)